crates/ruff_python_formatter/src/expression/expr_call.rs

```diff
@@ -10,11 +10,20 @@ use rustpython_parser::ast::ExprCall;
 pub struct FormatExprCall;
 
 impl FormatNodeRule<ExprCall> for FormatExprCall {
-    fn fmt_fields(&self, _item: &ExprCall, f: &mut PyFormatter) -> FormatResult<()> {
-        write!(
-            f,
-            [not_yet_implemented_custom_text("NOT_IMPLEMENTED_call()")]
-        )
+    fn fmt_fields(&self, item: &ExprCall, f: &mut PyFormatter) -> FormatResult<()> {
+        if item.args.is_empty() && item.keywords.is_empty() {
```

Comment:
We could also consider implementing this proper. We would only need to take care of dangling comments.